---
layout: default
title: Research
subtitle: Research in Jiang Lab
permalink: /research
---
We aim to have new scientific findings by developing statistical methods to summarize information from data and prior knowledege. Currently, we mainly focus on developing new statistical methods for (1) biological multi-omics data, such as (sc)RNA-seq, chip-seq, xWAS etc. to explore the mechanism underlying the complex phenotypes such as agronomic traits, diseases, tumors etc.; and (2) multimodal data including time series data, spatial data, network data etc. to integrate information from multiple dimensions to explore key features related to response.

### Research Interests 
- Computational Biology
    - personalized medicine (drug design and Target discovery)
    - genetic breeding (crossTalk between Crop and Environment)
    - Context-aware mechanisms of complex phenotypes
    - Statistical Genetics (Methods for Biobank data analysis)
- Statistical Methodology
    - Bayesian Modeling (MCMC)
    - Multimodal data modeling
    - Feature Selection
- Deep/Machine Learning
    - New Deep Learning Models

### Selected Publications 
#### [Note: \* denotes corresponding author(s), \# denotes joint first authors and authors with underline are (joint) students.](/resources)
#### 2024
1. <u>Mengting Shao</u>\#, <u>Min Tian</u>\#, <u>Kaiyang Chen</u>\#, **H.Jiang**\#,...., Chen Cao\*, Ning Gu\*. [Leveraging Random Effects in Cistrome-Wide Association Studies for Decoding the Genetic Determinants of Prostate Cancer](http://doi.org/10.1002/advs.202400815). ***Advanced Science***. DOI: 10.1002/advs.202400815
1. <u>M. Rui</u>, ....,Z. Chen, D. Tissue, **H.Jiang**\*, [Y. Wang\*](https://person.zju.edu.cn/wangyz). [Guard Cell and Subsidiary Cell Sizes are Key Determinants for Stomatal Kinetics and Drought Adaptation in Cereal Crops](https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.19757). ***New Phytologist***. DOI:10.1111/nph.19757

#### 2023
1. C. Cao\*, ...., **H.Jiang**, G. Wang\*, Q. Zhou\*. [RAVAR: a curated repository for rare variant-trait associations.](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkad876/7311081?login=true) ***Nucleic Acids Research***. DOI:10.1093/nar/gkad876
1. X. Feng\#, Z. Yu\#, H. Fang, **H.Jiang**, ....., Y. He\*,  F. Liu\*. [Plantorganelle Hunter is an effective deep-learning-based method for plant organelle phenotyping in electron microscopy](https://www.nature.com/articles/s41477-023-01527-5). ***Nature Plants***. DOI:10.1038/s41477-023-01527-5
1. Q. Ye\#, <u>Y. Li</u>\#,  H. Liu\#, J. Mao\*, **H.Jiang**\*. [Machine learning models for predicting steroid⁃resistant nephrotic syndrome](https://www.frontiersin.org/articles/10.3389/fimmu.2023.1090241/full) ([Webtool](https://datalinkx.shinyapps.io/srns/)). ***Frontiers in Immunology***. DOI:10.3389/fimmu.2023.1090241
1. J. Wang, X. Gu, ..., **H.Jiang**, B. Zheng, Y. Chen\*, Y. He\*. [Advances in the endogenous and exogenous regulation of anthocyanins–the key to color change in eudicots](https://www.tandfonline.com/doi/full/10.1080/07352689.2023.2227485). ***Critical Reviews in Plant Sciences***. DOI:10.1080/07352689.2023.2227485.
1. Q. Sun, ....., **H.Jiang**, D. Yan, B. Zheng, Y. He\*. [Transcriptome Analysis Reveals the Response Mechanism of Digitaria sanguinalis, Arabidopsis thaliana and Poa annua under 4,8-Dihydroxy-1-tetralone Treatment](https://www.mdpi.com/2223-7747/12/14/2728/review_report). ***Plants***. DOI:10.3390/plants12142728.
1. S. Qin,....., **H.Jiang**, ...., B. Zheng, Y. He\*. [Comparative Microscopic, Transcriptome and IAA Content Analyses Reveal the Stem Growth Variations in Two Cultivars Ilex verticillata](https://www.mdpi.com/2223-7747/12/10/1941). ***Plants***. DOI:10.3390/plants12101941.

#### 2022
1. <u>Q. Deng</u>\#, <u> H. Li</u>\#, Y. Feng\#, R. Xu, W. Li, R. Zhu, D. Akhter, X. Shen, J. Hu\*, **H.Jiang**\*, [R. Pan](https://person.zju.edu.cn/panr)\*. [Defining upstream enhancing and inhibiting sequence patterns for plant Peroxisome Targeting Signal type 1 using large-scale in silico and in vivo analyses.](https://onlinelibrary.wiley.com/doi/10.1111/tpj.15840) ***The Plant Journal***. DOI:10.1111/tpj.15840
1. **H.Jiang**, H. Li, [Wing H. Wong](https://biox.stanford.edu/people/wing-wong)\* and X. Fan\*. [Revealing Free Energy Landscape from MD Data via Conditional Angle Partition Tree](https://ieeexplore.ieee.org/abstract/document/9767706) [[code](/resources/capt.zip), [data](/resources/ala-traj.zip), [Supp](/resources/capt-supp.pdf)]. ***IEEE/ACM Transactions on Computational Biology and Bioinformatics***. DOI:10.1109/TCBB.2022.3172352.
1. <u>M. Xu</u>, Y. Chen, Z. Xu, L. Zhang\*, **H.Jiang**\*, C. Pian\*. [MiRLoc: predicting miRNA subcellular localizaition by incorporating miRNA-mRNA interaction and mRNA subcelluar localizaition](https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbac044/6532537). ***Briefings in Bioinformatics***. DOI:10.1093/bib/bbac044.
1. **H.Jiang**, [Xingqiu Zhao](https://www.polyu.edu.hk/ama/staff/zhaoxq/index.html), Ronald C.W. Ma and X.Fan\*. [Consistent Screening Procedures in High-dimensional Binary Classification](http://www3.stat.sinica.edu.tw/preprint/SS-2020-0088_Preprint.pdf). ***Statistica Sinica***. DOI: 10.5705/ss.202020.0088
1. <u>M. Rui</u>, Y. Jing, **H. Jiang**\*, [Y. Wang\*](https://person.zju.edu.cn/wangyz). [Quantitative System Modelling Bridges the Gap between Macro- and Microscopic Stomatal Model](https://onlinelibrary.wiley.com/doi/10.1002/adbi.202200131). ***Advanced Biology***. DOI: 10.1002/adbi.202200131.
1. J. Chen, Y. Huang, Y. Shu, X. Hu, D. Wu, **H. Jiang**, K. Wang, W. Liu and [W. Fu](https://person.zju.edu.cn/0019229)\*. [Recent progress on systems and synthetic biology of diatoms for improving algal productivity](https://www.frontiersin.org/journals/bioengineering-and-biotechnology/articles/10.3389/fbioe.2022.908804/full). ***Frontiers in Bioengineering and Biotechnology***. DOI: 10.3389/fbioe.2022.908804.

#### 2021
1. <u>H. Li</u>\#, H.X. Zhang\# and **H.Jiang**\*. [Combining Power of Different Methods to Detect Associations in Large Data Sets](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbab488/6447432) [[R package](/resources/DM.zip)]. ***Briefings in Bioinformatics***. DOI:10.1093/bib/bbab488
1. [H. Lei\#](https://person.zju.edu.cn/0018217), **H. Jiang**\#, N. Zhang, T. Chen, L. Yang, D. Wang and Y. Shu\*. [Increased urbanization reduced the effectiveness of school closures on seasonal influenza epidemics in China](https://idpjournal.biomedcentral.com/articles/10.1186/s40249-021-00911-7). ***Infectious Diseases of Poverty***. DOI:10.1186/s40249-021-00911-7
1. <u>Z. Li</u>\#, **H.Jiang**\#, L. Kong, Y. Chen, K. Lang, X. Fan, L. Zhang\* and C. Pian\*. [Deep6mA: a deep learning framework explores similar patterns in DNA N6-methyladenine sites across different species]( https://doi.org/10.1371/journal.pcbi.1008767). ***PLoS Computational Biology***. DOI:10.1371/journal.pcbi.1008767 (code: [Deep6mA](http://www.pianlab.cn/deep6ma/))
1. **H.Jiang** and X. Fan\*. [The Two-step Clustering Approach for Metastable States Learning](https://www.mdpi.com/1422-0067/22/12/6576). ***International Journal of Molecular Sciences***. DOI:10.3390/ijms22126576

#### 2020
1. Y. Chen\#, C. Kwok\#, **H.Jiang**\* and X.Fan\*. [Detect Differentially Methylated Regions Using Non-homogeneous Hidden Markov Model for Bisulfite Sequencing Data](https://doi.org/10.1016/j.ymeth.2020.09.009). ***Methods***. DOI:10.1016/j.ymeth.2020.09.009 ([code](/resources/BSDMR.zip))
1. **H.Jiang** and X.Fan\*. [A Consistent Variable Screening Procedure with Family-wise Error Control](https://doi.org/10.1080/00949655.2020.1724291). ***Journal of Statistical Computation and Simulation***. DOI:10.1080/00949655.2020.1724291
1. **H.Jiang**, [H.-b. Li\*](http://www.phy.cuhk.edu.hk/new/people/teaching/hbli.html) and X.Fan\*. [Bayesian Revisit of the Relationship between the Total Field Strength and the Volume Density of Interstellar Clouds](https://doi.org/10.3847/1538-4357/ab672b). ***The Astrophysical Journal***. DOI:10.3847/1538-4357/ab672b

#### Before ZJU
1. **H.Jiang** and Q.Wu\* (2018). [Robust dependence measure for detecting associations in large data set](https://doi.org/10.1016/S0252-9602(17)30117-0). ***Acta Mathematica Scientia***. DOI:10.1016/S0252-9602(17)30117-0
1. H.-B. Li\*, **H.Jiang** (in charge of statistical analysis), X.Fan, Q. Gu and Y. Zhang (2017). [The Link between Magnetic-field Orientations and Star Formation Rates](https://doi.org/10.1038/s41550-017-0158). ***Nature Astronomy*** (**Cover Paper**). DOI:10.1038/s41550-017-0158
1. Q.Tan, **H.Jiang**\* and Y.Ding (2014). [Model Selection Based on Maximal Information Coefficient of Residuals](https://doi.org/10.1016/S0252-9602(14)60031-X). ***Acta Mathematica Scientia***. DOI:10.1016/S0252-9602(14)60031-X
1. **H.Jiang**, Y.Shan and Q.Wu (2017). Dependece Measure: A comparative study. ***Acta Mathematica Scientia***.

#### Bookchapters
1. Q. Den, **H. Jiang**, J. Hu, R. Pan (2024). [Identification of auxiliary organellar targeting signals for plant peroxisomes using bioinformatic analysis of large protein sequence datasets followed by experimental validation](https://doi.org/10.1007/978-1-0716-3802-6). ***Photorespiration: Methods and Protocols***. Humana Press.

### Manuscripts 
1. Long-term barely diet alleviates cognitive deficits in AD mice via regulating gut microbiome composition.
1. Liquid-liquid phase separation-mediated regulation of crotonylation on the transcriptional networks of Streptomyces.
1. *Lactobacillus rhamnosus* improves lifestyle intervention response in Gestational Diabetes Mellitus impeded by Gut Bacterium *Bilophila wadsworthia*. G. Ye, ...., **H. Jiang**\*, Q. Luo\*.
1. Dual function of overexpressing plasma membrane H+-ATPase in balancing carbon-water use. **H. Jiang**\#, J. Su\#, ....., Y. Wang\*, Y. Wang\*.
1. [UK BioCoin: Swift Trait-Specific Summary Statistics Regression for UK Biobank.](https://biorxiv.org/cgi/content/short/2024.04.12.589273v1) <u>J. He</u>\#, <u>G. Q</u>\#, ...., **H. Jiang**\*, G. Chen\*. [(Homepage for UKC)](https://github.com/Ttttt47/UKBioCoin)
1.  [Case-specific selection of batch correction methods for integrating single-cell transcriptomic data from different sources.](https://biorxiv.org/cgi/content/short/2024.05.26.595911v1) X. Hu, ...., J. Qian\*, **H. Jiang**\*. ([R package of RBET](/resources/RBET_0.1.0.zip), [Homepage for RBET](https://github.com/zlyx26/RBET)).
1. Bioinspired cell silicification of the model diatom Phaeodactylum tricornutum and its effects on cell metabolism. Jiwei Chen, Cheng Qian, Yuexuan Shu, Kourosh Salehi-Ashtiani, Jin Shang, **H. Jiang**\*, Weiqi Fu\*.
1. CS-GAN: Commonality-Specificity Attention GAN for Weakly Supervised Medical Lesion Segmentation. G. Su, X. Zhao, J. Yin\*, **H. Jiang**\*.
1. [Non-parametric GWAS: Another View on Genome-wide Association Study.](https://www.biorxiv.org/content/10.1101/2022.11.11.516099v1) <u>X. Hu</u>\#, S. Yu\# and **H.Jiang**\*.
1. Bayesian Spatial-temporal Model for Detecting Alterations in Resting-State Functional Brain Network of Heavy Smokers, J. Hu, S. Chen, Z. Ye, F. Lin\* and **H.Jiang**\*.
1. Recent Advances on Phytohormone Regulation of Floral Bud Differentiation. X. Xie, S. Fu, L. Huang, S. Qin, J. Wang, Q. Sun, X. Wang, H. Yuan, D. Yan, Y. Wang, X. Mo, H. Wang, D. Xue, **H. Jiang**, J. Pozueta-Romero, B. Zheng, [Y. He](https://sky.zafu.edu.cn/info/1222/8304.htm)\*.
1. Non-parametric Gene-Enviroment interaction Detection, **H.Jiang** and X.Fan
1. [A Goodness-of-Fit Test for Statistical Models](https://arxiv.org/pdf/2006.08864.pdf), **H.Jiang**
1. [Bayesian Variable Selection for Single Index Logistic Model](https://arxiv.org/pdf/2012.06199.pdf), <u>Y. Sun</u> and **H.Jiang**\*.
1. Parameter Estimation in Noisy Dynamical System, **H.Jiang**, and  D.H., He
1. [Equitablity of Dependence Measure](https://arxiv.org/pdf/1501.02102.pdf), **H.Jiang**, K.Liu and Y.Ding (2013)
