---
layout: default
title: Software
subtitle: Softwares in Liu Lab
permalink: /software/
---

# Software
### Single Cell 
- [**MAESTRO**](http://github.com/liulab-dfci/MAESTRO) &nbsp;
Model-based Analyses of Transcriptome and Regulome (MAESTRO) is a comprehensive open-source computational workflow for integrative analysis of single-cell RNA-seq (scRNA-seq) and ATAC-seq (scATAC-seq) data from multiple platforms.
- [**TISCH**](http://tisch.comp-genomics.org/home/)  &nbsp; 
Tumor Immune Single-cell Hub (TISCH) provides detailed cell-type annotation for all curated tumor scRNA-seq datasets, enabling users to explore tumor immunity across many tumor scRNA-seq cohorts.

### Cancer Immunotherapy
- 
- [**TIMER**](https://cistrome.shinyapps.io/timer/) &nbsp; and latest version [**TIMER2**](http://timer.cistrome.org/) &nbsp;
Tumor IMmune Estimation Resource (TIMER) is a comprehensive resource for the clinical relevance of tumor-immune infiltrates, including B cells, CD4 and CD8 T cells, neutrophils, macrophages and dendritic cells.
- [**TRUST**](https://bitbucket.org/liulab/trust/) &nbsp; and latest version [**TRUST4**](https://github.com/liulab-dfci/TRUST4) &nbsp; 
is a computational algorithm that can extract TCR/BCR immune repertoires from tumor bulk or single cell RNA-seq data. 
- [**TIDE**](http://tide.dfci.harvard.edu/) &nbsp;
TIDE is a gene expression biomarker to predict the clinical response to immune checkpoint blockade. TIDE also provides gene-wise associations with T-cell dysfunction and immunotherapy resistance, computed from a large amount of cancer genomics datasets.

### Cancer Precision Medicine
- [**CARE**](http://care.dfci.harvard.edu/) &nbsp;
CARE is a software developed to enable large-scale inference of response biomarkers and drug combinations for targeted therapies using compound screen data.
- [**TIDE**](http://tide.dfci.harvard.edu/) &nbsp;
TIDE is a gene expression biomarker to predict the clinical response to immune checkpoint blockade. TIDE also provides gene-wise associations with T-cell dysfunction and immunotherapy resistance, computed from a large amount of cancer genomics datasets.
- [**DrugCellNet**](https://github.com/liulab-dfci/2LNet) &nbsp;
DrugCellNet is a dual-layer integrated cell line-drug network model, which uses both cell line similarity network (CSN) data and drug similarity network (DSN) data to predict the drug response of a given cell line using a weighted model.

### CRISPR Analysis
- [**MAGeCKFLute**](http://bioconductor.org/packages/release/bioc/html/MAGeCKFlute.html) &nbsp;
MAGeCKFlute is designed to support downstream analysis, utilizing the gene summary data provided through MAGeCK or MAGeCK-VISPR. Quality control, normalization, and screen hit identification for CRISPR screen data are performed in pipeline. Identified hits within the pipeline are categorized based on experimental design, and are subsequently interpreted by functional enrichment analysis.
- [**MAGeCK-VISPR**](http://bitbucket.org/liulab/mageck-vispr/) &nbsp;
Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout + Visualization of CRISPR Screens (MAGeCK-VISPR) is a comprehensive pipeline for analyzing and visualizing results from genome-scale CRISPR-Cas9 knockout screens. 
- [**MAGeCK**](https://bitbucket.org/liulab/mageck/src/master/) &nbsp;
Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identify important genes from genome-scale CRISPR-Cas9 knockout screens.
- [**NEST**](http://nest.dfci.harvard.edu/) &nbsp;
Network Essentiality Scoring Tool (NEST) is designed to predict gene essentiality based on protein interaction network and gene expression. It can also be used to enhance the quality of CRISPR screen results.  Publication submitted.
- [**SSC**](https://sourceforge.net/projects/spacerscoringcrispr/) &nbsp;
Sequence Scan for CRISPR gRNA efficiency.

### ChIP-Seq and Epigenetics Analysis
- [**LISA**](http://lisa.cistrome.org/) &nbsp;
leverages public ChIP-seq and chromatin accessibiity profiles in CistromeDB to discover the key transcription factors and chromatin regulators from user-provided differential expression gene list. 
- [**Cistrome-GO**](http://go.cistrome.org/) &nbsp;
Cistrome-GO is a webserver for functional enrichment analysis of transcription factor ChIP-seq peaks.
- [**Cistrome DB Toolkit**](http://dbtoolkit.cistrome.org/) &nbsp;
Backed by the huge collection of Cistrome DB, the Toolkit allows users to quickly examine their hypothesis regarding gene regulation through public ChIP-seq (protein factors and histone marks), chromatin accessibility (DNase-seq and ATAC-seq) data. Three functions have been implemented in Cistrome DB Toolkit website.
- [**RABIT**](http://rabit.dfci.harvard.edu/) &nbsp;
Regression Analysis with Background Integration (RABIT) is a very efficient feature selection algorithm. We applied RABIT to find tumor associated regulators in diverse cancer types. 
- [**Cistrome**](http://cistrome.org/ap/) &nbsp;
A web-based integrative platform for transcriptional regulation studies, from peak calling, correlation and association studies, expression analyses, to motif discovery.
- [**Cistrome DB**](http://cistrome.org/db) &nbsp;
Collects and processes public ChIP-seq / DNase-seq data for quality control, visualization, and download.
- [**Model-based Analysis of ChIP-Seq (MACS)**](https://github.com/taoliu/MACS) &nbsp;
A model-based algorithm for finding enriched regions in ChIP-Seq experiments.
- [**ChiLin**](http://cistrome.org/chilin/) &nbsp;
ChiLin is an extensible software suite that is ideal for processing and quality control of large batches of ChIP-seq and DNase-seq data. Publication submitted.
- [**BETA**](http://cistrome.org/BETA) &nbsp;
Targets analysis pipeline by integration of transcriptome and ChIP-seq data.
- [**BINOCh**](/resources/software/BINOCh.zip) &nbsp;
Binding Inference from Nucleosome Occupancy Changes.
- [**Nucleosome Positioning from Sequencing (NPS)**](/resources/software/NPS.zip) &nbsp;
A signal processing-based algorithm for identifying positioned nucleosomes fromsequencing experiments at the nucleosome level.
- [**Cis-regulatory Element Annotation System(CEAS) - Standalone version**](/resources/software/CEAS.zip) &nbsp;
A standalone Python package for analyzing ChIP-chip/seq data.
- [**Cis-regulatory Element Annotation System(CEAS)**](http://ceas.cbi.pku.edu.cn/) &nbsp;
An integrated webserver for analyzing ChIP-chip data.
 
### Transcriptome Analysis
- [**GFold**](https://bitbucket.org/feeldead/gfold) &nbsp;
Generalized fold change for ranking differentially expressed genes from RNA-seq data.
- [**SAGE-seq gene EXPRESSion analysis pipelines (sageExpress)**](/resources/software/sageExpress.zip) &nbsp;
Gene expression analysis pipelines for SAGE-Seq including tag mapping, novel normalization method using empirical Bayes and differential gene analysis.

### ChIP-chip Analysis
- [**Model-based Analysis of Tiling-array (MAT)**](/resources/software/MAT-3.07312009.tar.gz) &nbsp;
A model-based algorithm for finding enriched regions in ChIP-Chip experiments.
- [**Model-based Analysis of 2-Color Arrays (MA2C)**](/resources/software/MA2C.zip) &nbsp;
A model-based algorithm for analyzing 2-color microarrays.
- [**extreme MApping of OligoNucleotides (xMAN)**](/resources/software/xMAN.tar.gz) &nbsp;
A simple and efficient method for the mapping of millions of queryoligonucleotide fragments to the subject genome of any given length, at least an order of magnitude faster than other popular existing tools.
- [**Hidden Markov model for analyzing ChIP-chip experiments on genome tiling arrays (HMMTiling)**](/resources/software/HMMTiling.zip) &nbsp;
HMMTiling is a comprehensive software package for tiling array data analysis. It includes command line python applications for filtering, mapping, quantile-normalizing and enriched-region identification from ChIP-chip experiments on tiling arrays.
- [**Microarray Blob Remover (MBR)**](/resources/software/MBR.zip) &nbsp;
A microarray JAVA tool which allows rapid visualization, detection, and removalof blob-like defects as an initial quality control step.

### Transcription Factor Motif Finding
- [**CompareProspector**](http://compareprospector.stanford.edu/) &nbsp;
A sequence motif-finding algorithm.
- [**Motif Regressor**](http://www.math.umass.edu/~conlon/mr.html) &nbsp;
A program for finding DNA sequence motifs.
- [**MDscan**](http://robotics.stanford.edu/~xsliu/MDscan/) &nbsp;
A fast and accurate motif finding algorithm with applications to chromatin immunoprecipitation microarray experiments.
- [**BioProspector**](http://robotics.stanford.edu/~xsliu/BioProspector/) &nbsp;
Discovering conserved DNA motifs in upstream regulatory regions of co-expressedGenes.
